### PR TITLE
Switch to .NET Standard 1.0

### LINKED ISSUE #7: Move back to csproj-based project
So that this can be compiled with released versions of Visual Studio.

### PR DESCRIPTION
**Pull request by damieng**
Fixes #7 and #8

### CODE REVIEW

File: Microsoft.Linq.Translations/bin/Debug/netstandard1.0/Microsoft.Linq.Translations.deps.json [hunk 1]
```diff
@@ -0,0 +1,386 @@
+{
```
Comment (davidfowl) [inline Microsoft.Linq.Translations/bin/Debug/netstandard1.0/Microsoft.Linq.Translations.deps.json]: Need a better git ignore. This is in the bin folder

File: Microsoft.Linq.Translations.userprefs [hunk 2]
```diff
@@ -0,0 +1,12 @@
+﻿<Properties>
```
Comment (davidfowl) [inline Microsoft.Linq.Translations.userprefs]: Remove

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+language: csharp
```
Comment (davidfowl) [inline .travis.yml]: Should this be working?

Comment (damieng) [inline .travis.yml]: I took it from here... https://rimdev.io/building-net-core-on-travis-ci/

Comment (divega) [inline .travis.yml]: It seems you need to [add the build.sh file](https://rimdev.io/building-net-core-on-travis-ci/#buildsh).

**Pull request merged**